app/models/concerns/email_regex.rb

```diff
@@ -0,0 +1,28 @@
+module EmailRegex
```

Comment:
Two things:
- Modules should be used for mixins. This case is an isolated logic task which should be a class.
- `EmailRegex` is a to generic name for this specific use case. Please use the namespace `Channel::Filter::Match::EmailRegex` instead.

Reply:
Yeap. i will namespace that module. will see the new updated code soon.